geocolor/plugins/modules/algorithms/visir/GeoColor.py

```diff
@@ -248,7 +261,8 @@ def call(xobj):
     )
 
     # Normalize
-    sunzen = 1.0 - normalize(sunzen, min_sunzen, max_sunzen)
+    # Why are we raising to power of 1.5? It was in the GeoColor IDL counterpart
+    sunzen = (1.0 - normalize(sunzen, min_sunzen, max_sunzen)) ** 1.5
```

Comment:
I'm 99% sure that this is now double-applying the `** 1.5`, though it might be faster to do it here. This operation is also done on lines 326, 330, and 334.